graph/pull_v2_test.go

```diff
@@ -167,28 +167,4 @@ func TestValidateManifest(t *testing.T) {
 	if err == nil || !strings.HasPrefix(err.Error(), "image verification failed for digest") {
 		t.Fatal("expected validateManifest to fail with digest error")
 	}
-
-	// Manifest with no signature
-
-	expectedWholeFileDigest := "7ec3615a120efcdfc270e9c7ea4183330775a3e52a09e2efb194b9a7c18e5ff7"
```

Comment:
That is the definition of a signed manifest. If there is no signature, it is not valid. Rather than removing the test, can we change it to make it work correctly?